expyfun/_version.py

```diff
@@ -1 +1,2 @@
 __version__ = '2.0.0.dev0'
+__fork__ = 'ilogue'
```

Comment:
don't change this file manually.  in the repo it only has `__version__` defined as `dev0`, but when you run `setup.py` or `expyfun.download_version()` it will get automatically altered to include a commit hash at the end of the version string, and in the case of `download_version()` will also add the __fork__ variable.